modules/core/src/smithy4s/Service.scala

```diff
@@ -47,12 +48,16 @@ trait Service[Alg[_[_, _, _, _, _]], Op[_, _, _, _, _]] extends Transformable[Al
 
   def transform[P[_, _, _, _, _]](transformation: Transformation[Op, P]): Alg[P]
 
-  def asTransformation[F[_]](impl: Monadic[Alg, F]): Interpreter[Op, F] = asTransformationGen[GenLift[F]#λ](impl)
```

Comment:
If we're gonna break compat and bump minor, I want to remove this one which is biased towards `F[_]` kinds. Ideally, I want the bias to be centralised in the package object, so that we can easily offer some customisability there without impacting the main code generation template.